test/jdk/java/awt/a11y/AccessibleJComboboxTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2021, JetBrains s.r.o.. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8264287
+ * @summary Create implementation for NSAccessibilityComboBox protocol peer
+ * @author Artem.Semenov@jetbrains.com
+ * @run main/manual AccessibleJComboboxTest
```

Comment:
These tests can be applied wherever a11y support is implemented and running.

Reply:
Even on Linux? I meant if the test will be run on the platform where a11y is not implemented then this will be reported as a bug. So if on some supported platforms these tests will fail it is better to update the "tags" or the problemlist before integration. Otherwise, people will report new bugs after.

Reply:
Done